README.rst

```diff
@@ -134,6 +134,28 @@ Your buildout should be ready to use now.
 See `the documentation of mr.bob <http://mrbob.readthedocs.org/en/latest/>`_  for further information.
 
 
+Integration with plone.recipe.codeanalysis
+------------------------------------------
+
+This package is integrated by default with ``plone.recipe.codeanalysis`` that enforces the use of good known practices and the Plone code style guide.
+
+By default, the recipe setup a git precommit hook that performs the checks on the current package code. If it fins some violation the commit is not performed and the violations are shown.
```

Comment:
That's not exactly true, even if there are violations the commit happens, you could force that though, but is not on by default (I  guess with the ``return-status-codes = True``).